test-validator/src/lib.rs

```diff
@@ -541,6 +551,21 @@ impl TestValidator {
             genesis_config.ticks_per_slot = ticks_per_slot;
         }
 
+        // Remove features tagged to deactivate
+        for deactivate_feature_pk in &config.deactivate_feature_set {
+            if FEATURE_NAMES.contains_key(deactivate_feature_pk) {
+                match genesis_config.accounts.remove(deactivate_feature_pk) {
+                    Some(_) => info!("Feature for {:?} deactivated", deactivate_feature_pk),
+                    None => warn!(
+                        "{:?} account not found in genesis_config",
+                        deactivate_feature_pk
+                    ),
+                }
+            } else {
+                warn!("{:?} is not a feature public key", deactivate_feature_pk);
```

Comment:
Can we update these strings to better reflect that they're associated with feature deactivation?

Reply:
Pushed in [ae2428a](https://github.com/solana-labs/solana/pull/22860/commits/ae2428a9bba04cbaa5339269e9f343877d5fa9f0)